bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -484,9 +484,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm5_1" >/glade/u/home/slevis/paramfiles/ctsm51_params.c231117.nc</paramfile>
-<paramfile phys="clm5_0" >/glade/u/home/slevis/paramfiles/clm50_params.c231117.nc</paramfile>
-<paramfile phys="clm4_5" >/glade/u/home/slevis/paramfiles/clm45_params.c231117.nc</paramfile>
+<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c231117.nc</paramfile>
+<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c231117.nc</paramfile>
```

Comment:
If the Meier parameters aren't being added to the clm50 and clm45 parameter files -- why are they being updated?

Reply:
I updated the old params files to address issue #2247, which involved moving a few snicar parameters from the code to the params files.

Reply:
Perfect. Resolving.